### DISCUSSION TITLE
Fatal error when adding policies as an int

### DISCUSSION THREAD

**Discussion by lvanopstal** [quaquel/EMAworkbench#271] [Q&A] [answered]
Hello,

Currently I am coupling a Vensim model to the EMA workbench to do some analyses for my master thesis. I am running into an error and was wondering if this is the right place to ask what might be going wrong. After having defined uncertainties and outcomes and running the "evaluator.perform_experiments(1000, policies=5)" I get the following error (see attached picture).

EMAError: some fatal error has occurred while running the experiments, not all runs have completed. expected 5000, got 0

![Scherm­afbeelding 2023-05-18 om 14 02 46](https://github.com/quaquel/EMAworkbench/assets/83226570/1d67cfe7-8df9-4146-b833-d0ebdbaf4e8a)

I tried diving into the code snippet where the error gets thrown but couldn't figure out what I am doing wrong here. What could the problem be?

Thanks in advance!

**Comment by quaquel** [comment]
Impossible to answer based on the code shown. The error is a catchall for fatal errors. I need to see the complete code to judge this. 

Also, have you checked https://emaworkbench.readthedocs.io/en/latest/best_practices.html and made sure to apply this?

**Reply by lvanopstal** [reply]
I am currently running the code through jupyter notebooks. Even without the multiprocessing I get this error. The project directory looks similar as proposed in the link you provided. I hope this clarifies some things.  

![Scherm­afbeelding 2023-05-18 om 14 26 48](https://github.com/quaquel/EMAworkbench/assets/83226570/6733fc81-10e9-4a42-843e-e6cfcfe3ba12)

**Reply by quaquel** [reply, marked as answer]
A few remarks
1. why are you sampling 5 policies while you have not defined any levers? Most likely, this is the root of your error.
2. the r before the strings of wd and model_file is unnecessary. 
3. parallel processing should work even in jupyter, but I would still strongly advise against performing experiments in a notebook.

**Reply by lvanopstal** [reply]
Thanks for your reply. I looked into the mentioned points and this resolved the problem. Thank you!
